src/randomStrategy.ts

```diff
@@ -0,0 +1,61 @@
+import {Position} from './position'
+ 
+export class RandomStrategy {
+    constructor() {}
+    public getNextTarget(pos: Position, triedMap: { [ pos: string] : boolean }, hitMap: { [ pos: string] : boolean }): Position  {
+        //first shoot at position that have 2 neighbours already hit
+        var iterPos = new Position("A", 1);
+        while(iterPos.row != "J" || iterPos.column != 10) {
+            var positionString = iterPos.getString();
+            if(!triedMap[positionString] && iterPos.hasTwoNeighbours(hitMap)){
+                return iterPos;
+            }
+            iterPos = iterPos.getNextPosition();
+        }
```

Comment:
I would personally have each step of the strategy being a separate method, and setting the `iterPos` as a private class variable - it would cut some of the length of this class!